exampleSite/data/subsites_custom_logo.yml

```diff
@@ -0,0 +1,11 @@
+items:
+  -
+    page_url: '/subsite-1'
```

Comment:
@flora8984461 to confirm, this new logo will be used on sub pages of that section right?

Reply:
Yes, subsite-1, and its children, subsite-1/*/* ...